docs/deploy.md

```
@@ -0,0 +1,282 @@
+
+### 1-1. プロジェクト作成
+
+1. [Supabase](https://supabase.com) でプロジェクトを作成
+2. リージョン・パスワードを設定
+
+### 1-2. 接続 URL の取得
+
+**Settings → Database → Connection string** を開く。
+
+- **Transaction Pooler**（サーバーレス用・Cloud Run / Cloudflare 向け）
+  ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)